rotkehlchen/chain/ethereum/modules/makerdao/decoder.py

```diff
@@ -477,6 +552,16 @@ def decode_saidai_migration(self, context: DecoderContext) -> DecodingOutput:
 
     # -- DecoderInterface methods
 
+    def possible_events(self) -> DecoderEventMappingType:
```

Comment:
Can be done in another PR, but we need to make sure all events that this decoder are in such method